src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -1094,19 +1094,31 @@ public static void insertConfigIntoMPConfigPropertiesFile(RemoteRobot remoteRobo
      * Inserts a configuration name value pair into a config file via text typing
      * and popup menu completion (if required)
      *
-     * @param remoteRobot              The RemoteRobot instance.
-     * @param fileName                 The string path to the config file
-     * @param configNameSnippet        the portion of the name to type
-     * @param configNameChooserSnippet the portion of the name to use for selecting from popup menu
-     * @param configValueSnippet       the value to type into keyboard - could be a snippet or a whole word
-     * @param completeWithPopup        use popup to complete value selection or type in an entire provided value string
+     * @param remoteRobot                   The RemoteRobot instance.
+     * @param fileName                      The string path to the config file
+     * @param configNameSnippetCaseSpecific the portion of the name to type
+     * @param configNameChooserSnippet      the portion of the name to use for selecting from popup menu
+     * @param configValueSnippet            the value to type into keyboard - could be a snippet or a whole word
+     * @param completeWithPopup             use popup to complete value selection or type in an entire provided value string
      */
-    public static void insertConfigIntoConfigFile(RemoteRobot remoteRobot, String fileName, String configNameSnippet, String configNameChooserSnippet, String configValueSnippet, boolean completeWithPopup) {
+    public static void insertConfigIntoConfigFile(RemoteRobot remoteRobot, String fileName, String configNameSnippetCaseSpecific, String configNameChooserSnippet, String configValueSnippet, boolean completeWithPopup) {
         ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(30));
         clickOnFileTab(remoteRobot, fileName);
         EditorFixture editorNew = remoteRobot.find(EditorFixture.class, EditorFixture.Companion.getLocator());
         Exception error = null;
 
+        String configNameSnippet = "";
+        if (fileName.equals("server.env")) {
+            configNameSnippet = configNameSnippetCaseSpecific.toUpperCase();
+        }
+        else if (fileName.equals("bootstrap.properties")) {
+            configNameSnippet = configNameSnippetCaseSpecific.toLowerCase();
+        }
+        else {
+            configNameSnippet = configNameSnippetCaseSpecific;
+        }
+        String configValueSnippetUpperCase = configValueSnippet.toUpperCase();
```

Comment:
See https://github.com/OpenLiberty/liberty-tools-intellij/issues/1092 that was resolved last year.

```suggestion
        if (fileName.equals("server.env")) {
            configNameSnippet = configNameSnippetCaseSpecific.toUpperCase(java.util.Locale.ROOT);
        }
        else if (fileName.equals("bootstrap.properties")) {
            configNameSnippet = configNameSnippetCaseSpecific.toLowerCase(java.util.Locale.ROOT);
        }
        else {
            configNameSnippet = configNameSnippetCaseSpecific;
        }
        String configValueSnippetUpperCase = configValueSnippet.toUpperCase(java.util.Locale.ROOT);
```